src/sdc11073/pmtypes.py

```diff
@@ -1374,3 +1373,40 @@ class PatientType(StringEnum):
     INFANT = 'Inf'
     NEONATAL = 'Neo'
     OTHER = 'Oth'
+
+
+class CalibrationResult(PropertyBasedPMType):
+    Code = cp.SubElementProperty([namespaces.domTag('Code')], valueClass=CodedValue)
+    Value = cp.SubElementProperty([namespaces.domTag('Value')], valueClass=Measurement)
+    _props = ('Code', 'Value')
+
+
+class CalibrationDocumentation(PropertyBasedPMType):
+    Documentation = cp.SubElementListProperty([namespaces.domTag('Documentation')], cls=LocalizedText)
+    CalibrationResult = cp.SubElementListProperty([namespaces.domTag('CalibrationResult')], cls=CalibrationResult)
+    _props = ('Documentation', 'Value')
+
+
+class ComponentCalibrationState(StringEnum):
+    NOT = 'No'  # No = Not Calibrated. Defines that a component is not calibrated.
```

Comment:
T_ Prefix
Use same enum names as in master branch

Reply:
done